app/v2/controllers/requestParsers/validators/validations/ConsolidatedExpensesValidation.scala

```diff
@@ -90,4 +91,15 @@ object ConsolidatedExpensesValidation {
         }
     }
   }
+
+  def validate(expenses: UkNonFhlPropertyExpenses, path: String): List[MtdError] = {
+    expenses.consolidatedExpenses match {
+      case None => NoValidationErrors
+      case Some(_) =>
+        expenses match {
+          case UkNonFhlPropertyExpenses(None, None, None, None, None, None, None, None, None, None, Some(_)) => NoValidationErrors
```

Comment:
you may want to put `NoValidationErrors` and `List(...)` each on a separate line, as the auto-formatting goes a bit weird otherwise

Reply:
(& probably make the other matches in this class the same for consistency)